certified-connectors/Encodian/apiDefinition.swagger.json

```diff
@@ -3740,6 +3892,39 @@
         "description": "Provides the dynamic schema for the Convert CAD action."
       }
     },
+    "/api/v1/DynamicSchema/GetDynamicSchemaSignPdf": {
+      "get": {
+        "tags": [
+          "DynamicSchema"
+        ],
+        "operationId": "GetDynamicSchemaSignPdf",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "query",
+            "name": "certificateType",
+            "type": "string",
+            "x-ms-summary": "Certificate Type",
+            "description": "The output format to obtain the dynamic schema for."
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": { }
+          },
+          "400": {
+            "description": "Bad request",
+            "schema": { }
```

Comment:
@Encodian empty schema is not allowed, please define a proper response schema.

Reply:
@Amjed-Ayoub - The schema response is dynamic hence why it is not set. We have several dynamic schema actions all which follow this convention (/api/v1/DynamicSchema/GetDynamicSchemaConvertCad) as an example. The GetDynamicSchemaConvertCad action and others have been previously approved. I would request this is also approved as it matches previously approved convention.